R/pkg/R/mllib.R

```diff
@@ -51,13 +45,12 @@ setClass("PipelineModel", representation(model = "jobj"))
 #' summary(model)
 #'}
 setMethod("glm", signature(formula = "formula", family = "ANY", data = "DataFrame"),
-          function(formula, family = c("gaussian", "binomial"), data, lambda = 0, alpha = 0,
-            standardize = TRUE, solver = "auto") {
+          function(formula, family = c("gaussian", "binomial", "poisson", "gamma"), data,
+              lambda = 0, solver = "irls") {
```

Comment:
I would keep the solver to 'auto', so that we can change the implementation of the solver without regression. However, the option 'irls' is available for users who want to use it.

As for the other options for the solver, see my comment in the ticket.